src/pages/my_page/user_profile/MyProfile.tsx

```diff
@@ -36,27 +40,42 @@ export const MyProfile = () => {
     queryFn: () => fetchPerson(personId),
     onError: () => dispatch(setNotification({ message: t('feedback.error.get_person'), variant: 'error' })),
   });
-  const person = personQuery.data;
 
-  const hasActiveEmployment = filterActiveAffiliations(person?.affiliations).length > 0;
+  const person = personQuery.data;
 
   const firstName = getValueByKey('FirstName', person?.names);
   const lastName = getValueByKey('LastName', person?.names);
   const personPreferredFirstName = getValueByKey('PreferredFirstName', person?.names);
   const personPreferredLastName = getValueByKey('PreferredLastName', person?.names);
-  const [editPreferredNames, setEditPreferredNames] = useState(false);
   const personTelephone = person?.contactDetails?.telephone;
+  const [editPreferredNames, setEditPreferredNames] = useState(false);
+
+  const typographyStyling = {
```

Comment:
denne initieres på nytt for hver rerender nå, selv om den aldri endrer seg. Foreslår å flytte den over komponenten.

Ett annet forslag også; enten legg til type `SxProps` så vi vet at data her er riktig, eller lag en `const StyledTypography = styled({...})`. Foretrekker kanskje sistnevnte